<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1,
                                  stop: 0 #2c3e50, stop: 1 #3498db);
</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnNivel1">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #e74c3c;
    color: white;
    border: none;
    border-radius: 10px;
    font-size: 16px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #c0392b;
}
QPushButton:pressed {
    background-color: #a93226;
}</string>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Nivel 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnNivel2">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #f39c12;
    color: white;
    border: none;
    border-radius: 10px;
    font-size: 16px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #e67e22;
}
QPushButton:pressed {
    background-color: #d35400;
}</string>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Nivel 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnNivel3">
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #27ae60;
    color: white;
    border: none;
    border-radius: 10px;
    font-size: 16px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #229954;
}
QPushButton:pressed {
    background-color: #1e8449;
}</string>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Nivel 3</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #34495e; color: white;</string>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">background-color: #34495e; color: white;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>